### DISCUSSION TITLE
How do I get my clickevent to work

### DISCUSSION THREAD

**Discussion by scottvaughn5** [TroyAlford/react-jsx-parser#202] [Q&A]
Hi, 
I hope this is the right place for questions. I'm trying to write a sandbox for my team to play with. I'm having a really hard time getting my events to fire. Can someone help me figure out what it is that I'm doing wrong?

Parser component
```ts
<JsxParser
  bindings={{
    events:{ log: () => { console.log("Event") } },
    state: stateObject,
  }}
  components={{Button}}
  jsx={sandbox}
/>
```

String
```ts
<h1>Counter: {state.count}</h1>
<Button  onClick={events.log}>Hello World</Button>
```

**Comment by TroyAlford** [comment]
@scottvaughn5 - at the moment, I believe the bindings don't honor child elements. So binding to `events.log` won't work, but this would:
```ts
bindings={{
  log: () => { console.log('Event') },
}}
// then
<Button onClick={log}>Hello World</Button>
```
